law.cfg

```diff
@@ -10,7 +10,6 @@ columnflow.tasks.ml
 columnflow.tasks.union
 columnflow.tasks.histograms
 columnflow.tasks.plotting
-columnflow.tasks.inference
```

Comment:
Not an expert on `law` configs, sorry :D Why is this just gone? Does this mean that the CMS inference task isn't available anymore?

Reply:
As far as I understand, the inference tasks could not be called from the default cf setup, but you can simply add that module to the `[modules]` in your own analysis-specific law.cfg

Reply:
Yup, that would be the way to go :)